framework/src/postprocessors/PerfGraphData.C

```diff
@@ -20,44 +17,27 @@ InputParameters
 PerfGraphData::validParams()
 {
   InputParameters params = GeneralPostprocessor::validParams();
-
-  MooseEnum data_type("SELF CHILDREN TOTAL SELF_AVG CHILDREN_AVG TOTAL_AVG SELF_PERCENT "
-                      "CHILDREN_PERCENT TOTAL_PERCENT CALLS");
+  params.addClassDescription(
+      "Retrieves performance information about a section from the PerfGraph.");
 
   params.addRequiredParam<std::string>("section_name", "The name of the section to get data for");
-
   params.addRequiredParam<MooseEnum>(
-      "data_type", data_type, "The type of data to retrieve for the section_name");
-
-  params.addClassDescription("Retrieves timing information from the PerfGraph.");
+      "data_type", PerfGraph::dataTypeEnum(), "The type of data to retrieve for the section_name");
+  params.addParam<bool>("must_exist", true, "foo");
```

Comment:
I'm not sure that `foo` is the best documentation for this!

Reply:
oops